src/views/Swap.vue

```diff
@@ -239,6 +243,8 @@ const namespace = 'swap';
     TokenSelectButton: lazyComponent(Components.TokenSelectButton),
     TokenAddress: lazyComponent(Components.TokenAddress),
     ValueStatusWrapper: lazyComponent(Components.ValueStatusWrapper),
+    SwapTransactionDetails: lazyComponent(Components.SwapTransactionDetails),
+    InfoLine: components.InfoLine,
```

Comment:
Where we use info line in this component?

Reply:
fixed